src/backend/direct.rs

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::workldr::Workldr;
+
+use std::sync::{Arc, RwLock};
+
+use anyhow::{bail, Result};
+
+pub struct NilWorkldr;
```

Comment:
Let's be consistent about naming and rename `Nil` to `Direct`. Or maybe we should call this entire backend the "Wasm" backend and use that instead?

Reply:
But it is an empty Exec

Reply:
Workldr might be renamed to Exec and this to NilExec or NullExec

Reply:
ah..  name bikeshedding 😀